docs/controllers/services/examples/http-with-sticky-sessions.yml

```diff
@@ -36,3 +39,12 @@ spec:
         ports:
         - containerPort: 80
           protocol: TCP
+      # This is necessary for sticky-sessions because it can
+      # only consistently route to the same nodes, not pods.
+      affinity:
+        podAntiAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+          - labelSelector:
+              matchLabels:
+                app: hello
```

Comment:
Wouldn't this need to say `app: nginx-example` because that's what the label is like in line 34?

Reply:
Nice catch, was using the `hello` app to test. Updated :+1: